uid-pages/page-admin-profile-list/src/adminProfileListBonitaV1.json

```diff
@@ -14013,6 +14018,15 @@
         "    isAddMembershipGroupSearchMore: function(item) {",
         "        return item.name === uiTranslate(\"Or type more...\");",
         "    },",
+        "    isAddMembershipRoleSearchMore: function(item) {",
+        "        return item.name === uiTranslate(\"Or type more...\");",
+        "    },",
+        "    getGroupDisplayName(group) {",
+        "      return group.displayName ? group.displayName : group.name;  ",
+        "    },",
+        "    getRoleDisplayName(role) {",
```

Comment:
Why do you have the function twice, also, the function is not useful

Reply:
Same as mentioned above.